documents/final-submission-document/chapters/3_concept.tex

```diff
@@ -12,4 +12,6 @@ \section{Architecture}
 
 \section{Wireframes}
 
-
+We created Wireframes for most screens to visualise the initial set of user stories.
```

Comment:
@raphiz did you write "Wireframes" with a capital letter on purpose?

Reply:
One could argue, that this is a name for a concept...I have no strong opinion on this topic - if you want, i can change it...

Reply:
I changed it to lowercase, as [this seems to be a normally used word](https://en.oxforddictionaries.com/definition/wireframe).